catalog/src/main/kotlin/com/adevinta/spark/catalog/configurator/samples/bottomsheet/BottomSheetConfigurator.kt

```diff
@@ -0,0 +1,286 @@
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.unit.dp
+import com.adevinta.spark.SparkTheme
+import com.adevinta.spark.catalog.model.Configurator
+import com.adevinta.spark.catalog.util.SampleSourceUrl
+import com.adevinta.spark.components.bottomsheet.SheetState
+import com.adevinta.spark.components.bottomsheet.SheetValue
+import com.adevinta.spark.components.bottomsheet.modal.ModalBottomSheet
+import com.adevinta.spark.components.bottomsheet.modal.rememberModalBottomSheetState
+import com.adevinta.spark.components.buttons.ButtonFilled
+import com.adevinta.spark.components.icons.Icon
+import com.adevinta.spark.components.image.Illustration
+import com.adevinta.spark.components.image.Image
+import com.adevinta.spark.components.list.ListItem
+import com.adevinta.spark.components.menu.DropdownMenuItem
+import com.adevinta.spark.components.spacer.VerticalSpacer
+import com.adevinta.spark.components.text.Text
+import com.adevinta.spark.components.textfields.SelectTextField
+import com.adevinta.spark.components.toggles.SwitchLabelled
+import com.adevinta.spark.icons.LikeFill
+import com.adevinta.spark.icons.SparkIcons
+import com.adevinta.spark.icons.Store
+import kotlinx.coroutines.launch
+
+public val BottomSheetConfigurator: Configurator = Configurator(
+    name = "BottomSheet",
+    description = "BottomSheet configuration",
+    sourceUrl = "$SampleSourceUrl/BottomSheetSamples.kt",
+) {
+    BottomSheetSample()
+}
+
+@OptIn(ExperimentalMaterial3Api::class, ExperimentalFoundationApi::class)
+@Composable
+private fun ColumnScope.BottomSheetSample() {
+    var isDragHandlerEnabled by remember { mutableStateOf(true) }
+    var openBottomSheet by rememberSaveable { mutableStateOf(false) }
+    var skipPartiallyExpanded by remember { mutableStateOf(false) }
+    var bottomSheetContentExample by remember { mutableStateOf(BottomSheetContentExamples.Text) }
+
+    val bottomSheetState = rememberModalBottomSheetState(
+        skipPartiallyExpanded = skipPartiallyExpanded,
+        initialValue = SheetValue.Hidden,
+    )
+    val scope = rememberCoroutineScope()
+
+    Column(
```

Comment:
This Column is redondant since it's already set on the container configurator component